components/discover/claimModal.tsx

```diff
@@ -0,0 +1,213 @@
+    // TODO: Implement fetch from backend. Returning mock values.
+    try {
+      setLoading(true);
+      const rewards = [
+        {
+          appName: "EKUBO",
+          currencies: [
+            { currencyName: "STRK", value: 11570 }
+          ],
+        },
+        {
+          appName: "NOSTRA",
+          currencies: [
+            { currencyName: "STRK", value: 12.124 },
+            { currencyName: "ETH", value: 1.1245 }
+          ],
+        },
+        {
+          appName: "zkLend",
+          currencies: [
+            { currencyName: "USDT", value: 124.12 }
+          ],
+        },
+        {
+          appName: "VESU",
+          currencies: [
+            { currencyName: "STRK", value: 36 }
+          ],
+        },
+        {
+          appName: "Nimbora",
+          currencies: [
+            { currencyName: "STRK", value: 70.145 }
+          ],
+        },
+        {
+          appName: "zkLend",
+          currencies: [
+            { currencyName: "USDT", value: 124.12 }
+          ],
+        },
+        {
+          appName: "VESU",
+          currencies: [
+            { currencyName: "STRK", value: 36 }
+          ],
+        },
+        {
+          appName: "Nimbora",
+          currencies: [
+            { currencyName: "STRK", value: 70.145 }
+          ],
+        },
+      ];
+      const res = await new Promise<RewardItem[]>(resolve => setTimeout(() => resolve(rewards), 2000));
+      setClaimRewards(res);
+      setLoading(false);
+    } catch (error) {
+      setLoading(false);
+      showNotification("Error while fetching rewards", "error");
+      console.log("Error while fetching rewards", error);
+    }
+  }, []);
+
+  const doClaimRewards = useCallback(async () => {
+    // TODO: Implement logic to claim rewards
+    try {
+      setLoading(true);
+      await new Promise(resolve => setTimeout(resolve, 2000));
+      setLoading(false);
+      closeModal();
+      showSuccess();
+    } catch (error) {
+      setLoading(false);
+      showNotification("Error while claiming rewards", "error");
+      console.log("Error while claiming rewards", error);
+    }
+  }, []);
+
+  useEffect(() => {
+    if (open) {
+      getClaimRewards();
+    }
+  }, [open, getClaimRewards]);
+
+  return (
+    <Modal
+      disableAutoFocus
+      onClose={closeModal}
+      aria-label="Reward claim success modal"
+      open={open}
+    >
+      <div className={`${styles.popup} !overflow-y-hidden !rounded-2xl !mt-0 !px-1 !py-1 md:!px-0 md:!py-0`}>
+        <Loading isLoading={loading} loadingType="spinner">
+          <div className={`${styles.popupContent} !px-0 !py-2 md:!px-12 md:!py-6`}>
+            <div className="flex w-full flex-col self-start">
+              <div className="flex w-full lg:flex-row flex-col-reverse lg:items-start items-center justify-between gap-4">
+                <div className="flex flex-col gap-4 lg:text-left text-center">
+                  <Typography type={TEXT_TYPE.BODY_MIDDLE}>

```

Comment:
Could you add a bit of margin here on top of the text:
<img width="1036" alt="Screenshot 2024-10-11 at 3 09 13 PM" src="https://github.com/user-attachments/assets/51025f9c-9b40-4ed6-a762-7de1825d937c">
To look like the design on Figma:
<img width="685" alt="image" src="https://github.com/user-attachments/assets/ca28cf6c-d8fb-42e4-9999-a41fd5d69f4f">

Reply:
And also this one:
![image](https://github.com/user-attachments/assets/8eb27e3d-0835-47da-9fbf-340eaeb67c0b)